shared/actions/tracker.js

```diff
@@ -85,8 +85,8 @@ export function getProfile (username: string): TrackerActionCreator {
       return
     }
 
-    const trackerState = tracker.trackers[username] && tracker.trackers[username]
-    const uid = trackerState.type === 'tracker' ? trackerState.userInfo && trackerState.userInfo.uid : null
+    const trackerState = tracker && tracker.trackers ? tracker.trackers[username] : null
```

Comment:
Annoyingly if you don't do it like this flow won't infer the type correctly.

I think this is because the value coulde be null or tracker or tracker.trackers or tracker.trackers[username] and flow gives up.

With this change it will always be either `null` or `tracker.trackers[username]`